.editorconfig

```diff
@@ -0,0 +1,16 @@
+# top level config
+root = true
+
+# All files:
+# utf-8
+# Unix-style newlines
+# Remove end-of-file newlines
+# Soft indents (2 spaces)
+# Don't trim trailing whitespace
```

Comment:
Ah yes, all our text should be using locales strings, with variables when necessary, so this wouldn't occur, at least in an ideal world. :) I think there's still some text that's not translated but if we ran into that problem with whitespace, we'd fix it by making it translatable. Thanks for the explanation!

@MathWhiz Sorry, could you please remove the exception for jade/pug?